l2-contracts/contracts/bridge/L2SharedBridge.sol

```diff
@@ -58,6 +58,7 @@ contract L2SharedBridge is IL2SharedBridge, Initializable {
         require(_l2TokenProxyBytecodeHash != bytes32(0), "df");
 
         l1Bridge = _l1Bridge;
+        l1LegacyBridge = _l1LegecyBridge;
```

Comment:
I know that this typo hasn't been introduced with these changes, but can you fix it?